packages/commonwealth/client/scripts/views/pages/AdminPanel/UpdateCustomDomainTask.tsx

```diff
@@ -72,19 +73,6 @@ const UpdateCustomDomainTask = () => {
     if (!validCustomDomainUrl.test(customDomain)) {
       return 'Invalid URL (try removing the http prefix)';
     }
-
-    // there's probably a better way to remove prefixes for duplicate finding purposes
-    const existingCustomDomain = app.config.chains
```

Comment:
why we dont need that anymore?

Reply:
its moved to api
> Updated community update API to check for duplicate/already-taken custom domains and removed that check from the front end. Now when super admin is updating the custom domain of a community, we show duplicate/already-taken errors on form submit, rather than live as user types. This was needed as we don't store the whole chains/communities list on the front end anymore.